app/src/main/java/org/dhis2/usescases/teiDashboard/DashboardRepositoryImpl.java

```diff
@@ -374,12 +373,34 @@ public void setDefaultCatOptCombToEvent( String eventUid )
     {
         if ( programUid != null )
             return briteDatabase
-                .createQuery( ATTRIBUTE_VALUES_TABLE, ATTRIBUTE_VALUES_QUERY, programUid, teiUid == null ? "" : teiUid )
-                .mapToList( cursor -> ValueUtils.transform( briteDatabase, cursor ) );
+                    .createQuery( ATTRIBUTE_VALUES_TABLE, ATTRIBUTE_VALUES_QUERY, programUid, teiUid == null ? "" : teiUid )
```

Comment:
Could it change the query to modules from D2 ?

Reply:
For the time being, we decided not to change it to the sdk modules. This will be done in the near future.